fe2-android/app/src/main/res/layout/roll_call_fragment.xml

```diff
@@ -133,5 +133,31 @@
       app:layout_constraintStart_toStartOf="@id/roll_call_start_time"
       app:layout_constraintTop_toTopOf="@id/end_time_icon" />
 
+    <TextView
+      android:id="@+id/roll_call_attendees_text"
+      style="@style/explication_text_style"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="@string/roll_call_attendees"
+      app:layout_constraintBottom_toTopOf="@+id/list_view_attendees"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/roll_call_fragment_end_time_title" />
+    <ListView
+      android:id="@+id/list_view_attendees"
+      android:scrollbarStyle="outsideOverlay"
+      android:layout_width="@dimen/qr_rollcall_img"
+      android:layout_height="@dimen/qr_rollcall_img"
+      android:layout_marginEnd="10dp"
```

Comment:
Hardcoded values should be avoided. Though granted the app would be in need of some uniformization of `dimens` usage for the benefices to really show

Reply:
Sorry, completely missed that value! Going to fix that